src/main/java/org/symphonyoss/symphony/messageml/elements/Checkbox.java

```diff
@@ -89,9 +106,57 @@ public Node asMarkdown() {
     return new CheckboxNode();
   }
 
+  private void buildElementFromGroupDiv(MessageMLParser context, org.w3c.dom.Element element) throws InvalidInputException, ProcessingException {
+    NodeList children = element.getChildNodes();
+
+    if (children.getLength() != PRESENTATIONML_DIV_NUMBER_OF_CHILDREN) {
+      throw new InvalidInputException(String.format("Invalid PresentationML for the \"%s\" element", MESSAGEML_TAG));
```

Comment:
The error message could be a little more specific. It's not clear to the user why the content is invalid. And you could add a Javadoc describing what the expected input is.